### PR TITLE
795 - Replace `readme_file.get_file_contents` template with new generic `readme.md`

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number
Closing #795

## Purpose/Implementation Notes
This PR includes the following changes:
- Provided the following template contexts to the `readme.md`:
  -  `contents_template` for the contents section
  - `download_config` to conditionally render the usage section
- Cleaned up no longer needed unique readme template files per computed file type
- Adjusted the existing test to verify the readme content per computed file type 

## Types of changes

- Refactor (addresses code organization and design mentioned in corresponding issue)

## Functional tests

N/A

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots

N/A

### CODE REVIEW

File: api/scpca_portal/templates/readme/usage/index.md
Comment (davidsmejia) [inline api/scpca_portal/templates/readme/usage/index.md:1]: Let's create a `usage/index.md` file and then if modality is `SINGLE_CELL` we can render the section title and one of the nested templates. I think the checks should be in the following order:

- if format ann_data -> 
- elif excludes_multiplexed
- else (with multiplexed)

**Review by davidsmejia — COMMENTED** [review]
Almost there just need to tidy up the usage template.

File: api/scpca_portal/templates/readme/usage/index.md
Comment (nozomione) [reply api/scpca_portal/templates/readme/usage/index.md:1]: See the commit at 7aef3f03a61bd19f5771631867f021c4af8b362d.

Comment (nozomione) [reply api/scpca_portal/templates/readme/usage/index.md:1]: **NOTE:** I've also reduce the extra space in this file `readme/usage/index.md` at bdb1674a2043cf17a812ba3e2420e487c6859610.

Comment (nozomione) [reply api/scpca_portal/templates/readme/usage/index.md:1]: I've also reduce the extra space and remove `else` from this template, thank you David!

File: api/scpca_portal/templates/readme/usage/index.md [hunk 1]
```diff
@@ -1,19 +1,15 @@
-<!-- SINGLE_CELL_EXPERIMENT -->
 
+{% if download_config.modality == 'SINGLE_CELL' %}
```
Comment (davidsmejia) [inline api/scpca_portal/templates/readme/usage/index.md:2]: Let's remove the extra empty line at the top of this file.

**Review by davidsmejia — APPROVED** [review]
LGTM after removing that one extra empty line.

**Pull request merged**